.env.sample

```
@@ -31,11 +36,5 @@ AWS_SES_SECRET_ACCESS_KEY=
 
 OPENAI_API_KEY=
-# ============================================================================
-# Dev Patches (optional - for local development only)
-# ============================================================================
-# These variables enable dev-only features via the .dev patch system.
-# See .dev/README.md for details on available patches.
-#
-# DEV_BYPASS_AUTH=true     # Skip Google OAuth, use dev login menu instead
-# DEV_MOCK_STORAGE=true    # Use mock storage instead of R2/S3
+
+TOKEN_SIGNING_SECRET=CHANGE_ME
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The TOKEN_SIGNING_SECRET key is duplicated

(DuplicatedKey)